Implementation/Tabs with icon and svg/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>
+      Tabs with svg hover and active
+    </title>
+    <link rel="stylesheet" href="css/master.css">
+  </head>
+  <body>
+    <div class="swapper">
+      <a href="#" class="swp swapper-link">
+        News
+        <img class = "swapper-img" src ="icons/download.png" />
+      </a>
+      <li class="swp swappper-item swp-active">
+        Chat
+        <img class = "swapper-img" src="icons/chat.png" alt="">
+      </li>
+      <li class="swp swappper-item">
+        Videos
+        <img class = "swapper-img" src="icons/video.png" alt="">
+      </li>
+      <li class="swp swappper-item">
+        Photos
+        <img class = "swapper-img" src="icons/photos.png" alt="">
+      </li>
```

Comment:
Include them in a ul tag and for a more clear structure include Text inside anchor tag of each li and add proper classes.

Reply:
I have added the ul tag along with alt text and title. also images now appear more smotthly

Reply:
LGTM!